src/main/resources/messages_fr.properties

```diff
@@ -0,0 +1,73 @@
+# agreements. See the NOTICE file distributed with this work
+# for additional information regarding copyright ownership.
+# Jasig licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a
+# copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+feedback.form.question=Aimez-vous le portail ?
+feedback.form.suggestion=Proposer une suggestion, un commentaire...
+feedback.form.charactersremaining=Vous avez {0} caract\u00E8res restants.
+feedback.form.anonymous=Soumission anonyme
+feedback.form.submit=Soumettre
+
+feedback.answer.yes=oui
+feedback.answer.no=non
+feedback.answer.maybe=ne souhaite pas s'exprimer
+
+feedback.success.title=Merci!
+feedback.success.message=Merci d'avoir apport\u00E9 un retour d'utilisation du portail.  Votre commentaire a \u00E9t\u00E9 enregistr\u00E9 avec succ\u00E8s.
+feedback.success.more=Soumettre plus de retours d'utilisation
+
+feedback.empty.title=Pas de retours utilisables!
+feedback.empty.message=Vous n'avez pas laiss\u00E9 de retour. Vous devez soit saisir un commentaire ou s\u00E9lectionner "oui" ou "non" pour que votre retour soit utilisable.
+feedback.empty.more=Retourner \u00E0 la soumission d'un retour d'utilisation
+
+feedback.admin.title=Retours r\u00E9cents
+feedback.admin.stats.total=Total des r\u00E9ponses
+feedback.admin.stats.unique=Utilisateurs uniques
+feedback.admin.stats.answer.all=tous
+feedback.admin.stats.answer.yes=positif
+feedback.admin.stats.answer.no=n\u00E9gatif
+feedback.admin.stats.answer.maybe=ind\u00E9cis
+
+feedback.admin.type.all=tous
+feedback.admin.type.student=\u00E9tudiant
+feedback.admin.type.staff=personnel
+feedback.admin.type.faculty=enseignant
+
+feedback.admin.form.show=Montrer
+feedback.admin.form.role=R\u00F4le
+feedback.admin.form.type=Type
+feedback.admin.form.showing=Affichage de
+feedback.admin.form.of=sur
+feedback.admin.form.next=suivant
+feedback.admin.form.prev=pr\u00E9c\u00E9dent
+feedback.admin.form.submit=Rafra\u00EEchir
+feedback.admin.form.jumpto=Aller \u00E0 l'index
+feedback.admin.form.goto=ok
+
+feedback.admin.rowtitle.page=Page
+feedback.admin.rowtitle.name=Nom
+feedback.admin.rowtitle.role=R\u00F4le
+feedback.admin.rowtitle.browser=Navigateur
+feedback.admin.rowtitle.time=Date
+feedback.admin.anonymous=Anonyme
+feedback.admin.export=exporter
+
+feedback.admin.filter.title=Filtrer par date
+feedback.admin.filter.startdate=Date de d\u00E9but
+feedback.admin.filter.enddate=Date de fin
+feedback.admin.form.comments=Montrer seulement les retours avec des commentaires 
```

Comment:
Fantastic! :sparkles: 
That's what I thought, I just wanted a second pair of eyes :eyes: to take a look.
Thanks @cousquer and @auxepaul !